openshift-hack/images/os/Dockerfile

```diff
@@ -21,3 +21,6 @@ RUN set -x && yum install -y ostree rpm-ostree yum-utils selinux-policy-targeted
 
 FROM scratch
 COPY --from=build /srv/ /srv/
+
+LABEL io.openshift.build.version-display-names="machine-os=rhcos image for testing openshift kubernetes kubelet only.  If you see this outside of PR runs for openshift kubernetes- you found an urgent blocker bug" \
```

Comment:
using a `-` instead of a `,` to [avoid][1]:

```
...must only be alphanumerics, spaces, and symbols in [():-]
```

[1]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_kubernetes/963/pull-ci-openshift-kubernetes-master-images/1438857360410939392#1:build-log.txt%3A91